Minor fixes in concepts-multiple-qubits.md (#596)
MicrosoftDocs/quantum-docs: articles/concepts-multiple-qubits.md

    \end{bmatrix}.
$$

> It is also possible to measure just one qubit of a two-qubit quantum state. In such case, the impact is subtly different; only the related subsystem, not the entire state, is collapsed to a computational basis state.  

Reviewer: ```suggestion
It's also possible to measure just one qubit of a two-qubit quantum state. When you measure only one qubit of a two-qubit state, the impact of measurement is subtly different than measuring two qubits. This is because the entire state isn't collapsed to a computational basis state, rather it's collapsed to only one subsystem.  In other words, measuring one qubit of a two-qubit state only collapses the related subsystem to a computational basis state. 
```

Author: @SoniaLopezBravo I seriously don't understand from the context what exactly you mean by
> This is because the entire state isn't collapsed to a computational basis state, rather it's collapsed to only one subsystem.

But intuitively it sounds wrong to me. I would say "only one subsystem of the whole system gets collapsed", not "the whole system gets collapsed to only one subsystem". And the next sentence supports that.

And do you think that duplicated phrases like
> It's also possible to measure just **one qubit of a two-qubit quantum state**. When you measure only **one qubit of a two-qubit state** ...

are what makes the text more readable?

I though pull requests could be more about discussion to find a clear formulation. If I'm suggesting a change, it's because I as a new reader was disturbed by something in the formulation or had issues understanding it.

> [!NOTE]
> This article uses the *little-endian* format to label the computational basis. In little endian format, the least significant bits come first. For example, the number four in little-endian format is represented by the string of bits 001.

Since each qubit measurement can only yield $0$ or $1$, the probability of measuring $0$ is simply $1-P(\text{outcome}=1)$.  This is why you only need a formula for the probability of measuring $1$.

The action that such a measurement has on the state can be expressed mathematically as

$$
\psi \mapsto \frac{\sum_{e_k \text{ in the set } S} e_k e_k^\dagger \psi}{\sqrt{P(\text{outcome}=1)}}.
$$

> The cautious reader may worry about what happens if the denominator is zero.  While such state is undefined, it cannot occur because the probability is zero!

Reviewer: ```suggestion
The cautious reader may worry about what happens if the denominator is zero.  While such state is undefined, you don't need to worry about such eventualities because the probability is zero! 
```
